src/main/java/bio/terra/cli/businessobject/Workspace.java

```diff
@@ -145,14 +145,16 @@ private static Workspace convertWorkspaceDescriptionToWorkspace(
   /**
    * Update the mutable properties of the current workspace.
    *
+   * @param id optional display id
    * @param name optional display name
    * @param description optional description
    * @throws UserActionableException if there is no current workspace
    */
-  public Workspace update(@Nullable String name, @Nullable String description) {
+  public Workspace update(
+      @Nullable String id, @Nullable String name, @Nullable String description) {
```

Comment:
Somewhere we should check that one or more of these is non-null.

Reply:
Do you think i can write an assertion here(L155) and L262?

This is not a test, I think if I write an assertnotnull will be wired? Or just a if statement is fine?

Reply:
I'd do an `if` statement and throw a `UserActionableException`. There should be examples where we do this.